stock_quant_manual_assign/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Manual assignment of quants
+===========================
+
+With this module, user can change manually the automatic selection of quants
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
No, I'm telling that for this specific module, explain how to use it, and then add then put this same thing. We can improve the template adding something like:
```
Usage
=====

<Put here you usage instructions>

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/153/8.0
```
But it's not what I intend here.